MOBILE_API.xml

```diff
@@ -449,6 +449,7 @@
         <element name="APP_UNAUTHORIZED" since="2.0" />
         <element name="PROTOCOL_VIOLATION" since="4.0" />
         <element name="UNSUPPORTED_HMI_RESOURCE" since="4.1" />
+        <element name="RESOURCE_CONSTRAINT" since="6.2" />
```

Comment:
Please add a description like "The application is unregistered due to hardware resource constraints. The system will shortly close the application to free up hardware resources." to this element.

Reply:
Comment added